crates/model/src/solver_competition.rs

```diff
@@ -109,13 +109,30 @@ pub struct SolverSettlement {
     pub uninternalized_call_data: Option<Vec<u8>>,
 }
 
+#[serde_as]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(untagged)]
+pub enum Cost {
+    // estimated by the protocol
+    Protocol(f64),
+    // estimated by the solver
+    Solver(f64),
```

Comment:
Can we be extra explicit about what they are (ie ProtocolEstimated), otherwise it sounds like one is cost that the solver incurs and the other one is cost that the protocol incurs. Below as well